src/components/userFeedback/UserFeedback.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import { shallow, ShallowWrapper } from 'enzyme';
```

Comment:
Is there any reason why we would only use the shallow rendering here? In every other tests we are using the ReactWrapper and the mount function. Maybe we are doing it wrong! 😄